tests/python/unittest/test_higher_order_grad.py

```diff
@@ -18,8 +18,14 @@
 
 import math
 from mxnet import nd, autograd
-from mxnet.test_utils import assert_almost_equal, random_arrays, rand_shape_nd
+from mxnet.test_utils import assert_almost_equal, random_arrays, rand_shape_nd, same
 from common import with_seed
+import mxnet.autograd as ag
+import mxnet.ndarray as nd
+from mxnet import gluon
+import mxnet
+from nose.tools import ok_
```

Comment:
this module should be imported before mxnet module

Reply:
I believe you, but can you elaborate?

Reply:
https://www.python.org/dev/peps/pep-0008/#imports

Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.